packages/elements/src/tree/elements/tree.ts

```diff
@@ -425,8 +425,34 @@ export class Tree<T extends TreeDataItem = TreeDataItem> extends List<T> {
       return this.composer.getItemPropertyValue(item, 'value') as string || '';
     });
   }
-  public set values (value: string[]) {
-    super.values = value;
+  public set values (values: string[]) {
+    if (!Array.isArray(values)) {
+      valueFormatWarning.show();
+      this.values = [];
+    }
+    else {
+      // Clone value arrays
+      const newValue = values.slice();
```

Comment:
Why need to clone value array? you can use `const newComparison = values.sort().toString()` directly, Isn't it?

Reply:
You should not sort the real values.

Reply:
How about `const newComparison = [...values].sort().toString()` I'm not sure, what do you think?

Reply:
fixed